backend/services/syson-diagram-services/src/main/java/org/eclipse/syson/diagram/services/DiagramMutationDiagramService.java

```diff
@@ -87,4 +100,61 @@ public Element createDiagram(Element element, IEditingContext editingContext) {
         }
         return element;
     }
+
+    /**
+     * Duplicates the given element (with its owning Relationship) and exposes the duplicated element. Note that this method will do nothing on {@link org.eclipse.syson.sysml.Relationship}.
+     *
+     * @param element
+     *         the element to duplicate
+     * @param editingContext
+     *         the {@link IEditingContext}
+     * @param diagramContext
+     *         the {@link DiagramContext}
+     * @param node
+     *         the node that have been duplicated
```

Comment:
the node that have been duplicated? It's not "the node that will be duplicated"?